### PR TITLE
Added Finalist Entry

### PR DESCRIPTION
**Pull request by Manoj-M-S**
Hi Mentor,

Hope you are doing good, I have complete all 5 projects and 3 Blogs, Please review and accept my pull request. 

Thankyou.

### CODE REVIEW

File: content/finalist/ttjc@finalist@Manoj-M-S/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+### blogs
+
+1. [How to Fix useEffect Hook executing in an endless loop?] (https://dev.to/manojms/how-to-fix-useeffect-hook-executing-in-an-endless-loop-1oga) 
+  - This Blog is about how to Fix the React Hook, useEffect executing in an endless loop.
+
+2. [Create a basic React Project by implementing React-Router within 10 mins] (https://dev.to/manojms/create-a-basic-react-project-by-implementing-react-router-within-10-mins-pl5)
+  - This Blog is about creating basic React project, by implementing React-Router within 10 mins.
+
+3. [Basic Git Commands] (https://dev.to/manojms/basic-git-commands-jkp)
+  - This Blog is about Basic Git Commands.
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@Manoj-M-S/index.md:106]: provide a proper description and use proper markdown

Comment (Manoj-M-S) [reply content/finalist/ttjc@finalist@Manoj-M-S/index.md]: Hi Mentor,

Thankyou for your feedback, now I have made the required changes, please review and accept my pull request.

Thankyou.

Comment (gkanishk) [reply content/finalist/ttjc@finalist@Manoj-M-S/index.md]: still the same remove space in between!!!!!

**Comment by gkanishk** [comment]
Should check your PR before raising it, I requested to correct the markdown but you were unable to.

**Pull request closed without merging**